### PR TITLE
feat: update graphql documentation for new endpoints

### LINKED ISSUE #6133: [data node] New endpoints lack documentation
https://github.com/vegaprotocol/vega/blob/develop/datanode/gateway/graphql/schema.graphql

This should have docblocks for all queries. Frequently the new ones don't, where the old ones do:
https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L369-L377

# Tasks: Queries MUST have docs
- [x] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L359
- [x] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L372
- [x] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L408
- [x] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L602
- [x] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L657
- [x] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L874
- [x] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L886
- [ ] Add docs for https://github.com/vegaprotocol/vega/blob/develop/datanode/gateway/graphql/schema.graphql#L989
- [ ] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L874
- [ ] Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L886

## Optional extra (can be another ticket): Types should all have documentation
-  Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L710
-  Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L781
-  Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L791
-  Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L802
-  Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L826
-  Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L837

-  Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L842
-  Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L861
- Add docs for https://github.com/vegaprotocol/vega/blob/a046379e01699f378ac86538592e6cbf06d3c29a/datanode/gateway/graphql/schema.graphql#L896

### PR DESCRIPTION
**Pull request by guoguojin**
Closes #6133

### CODE REVIEW

File: datanode/gateway/graphql/schema.graphql [hunk 1]
```diff
@@ -474,11 +475,15 @@ type Query {
     "Optional Pagination"
     pagination: Pagination): MarketDataConnection
 
+  "get historical balances for an account or specific asset within the given date range."
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:478]:
```suggestion
  "Get historical balances for an account or specific asset within the given date range."
```

File: datanode/gateway/graphql/schema.graphql [hunk 2]
```diff
@@ -497,6 +502,7 @@ type Query {
   "One or more instruments that are trading on the VEGA network"
   markets("ID of the market" id: ID): [Market!] @deprecated(reason: "Use marketsConnection instead")
 
+  "One or more instruments that are trading on the VEGA network"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:505]:
```suggestion
  "One or more instruments that are trading on the Vega network"
```

File: datanode/gateway/graphql/schema.graphql [hunk 3]
```diff
@@ -516,6 +522,7 @@ type Query {
   "return the full list of network parameters"
   networkParameters: [NetworkParameter!] @deprecated(reason: "Use networkParametersConnection instead")
 
+  "return the full list of network parameters"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:525]:
```suggestion
  "Return the full list of network parameters"
```

File: datanode/gateway/graphql/schema.graphql [hunk 4]
```diff
@@ -778,16 +788,19 @@ type RecurringTransfer {
   dispatchStrategy: DispatchStrategy
 }
 
+"The type of metric to use for a dispatch strategy"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:791]:
```suggestion
"The type of metric to use for a reward dispatch strategy"
```

File: datanode/gateway/graphql/schema.graphql [hunk 5]
```diff
@@ -778,16 +788,19 @@ type RecurringTransfer {
...
 enum DispatchMetric {
+  "Dispatch metric that is using the total maker fees paid in the market"
   DISPATCH_METRIC_MAKER_FEES_PAID
-
+  "Dispatch metric that is using the total maker fees received in the market"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:795]:
```suggestion
  "Dispatch metric that uses the total maker fees received in the market"
```

File: datanode/gateway/graphql/schema.graphql [hunk 6]
```diff
@@ -778,16 +788,19 @@ type RecurringTransfer {
...
   DISPATCH_METRIC_MAKER_FEES_RECEIVED
-
+  "Dispatch metric that is using the total LP fees received in the market"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:797]:
```suggestion
  "Dispatch metric that uses the total LP fees received in the market"
```

File: datanode/gateway/graphql/schema.graphql [hunk 7]
```diff
@@ -778,16 +788,19 @@ type RecurringTransfer {
...
   DISPATCH_METRIC_LP_FEES_RECEIVED
-
+  "Dispatch metric that is using total value of the market if above the required threshold and not paid given proposer bonus yet"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:799]:
```suggestion
  "Dispatch metric that uses the total value of the market if it's above the required threshold and not paid the given proposer bonus yet"
```

File: datanode/gateway/graphql/schema.graphql [hunk 8]
```diff
@@ -799,6 +812,7 @@ type DispatchStrategy {
   marketIdsInScope: [ID!]
 }
 
+"Validating status of a node, i.e. Validator or non-validator"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:815]:
```suggestion
"Validating status of a node, i.e. validator or non-validator"
```

File: datanode/gateway/graphql/schema.graphql [hunk 9]
```diff
@@ -858,6 +875,7 @@ type EthereumKeyRotation {
   blockHeight: String!
 }
 
+"Epoch describes a specific period of time in the Vega network"
 type Epoch {
   "Presumably this is an integer or something. If there's no such thing, disregard"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:880]:
```suggestion
  "Presumably this is an integer or something. If there's no such thing, disregard"
```
Not part of the PR I know ... but might there be a better description for this?

**Review by candida-d — COMMENTED** [review]
All very small suggested changes. One request for a better description, if possible :)

File: datanode/gateway/graphql/schema.graphql (hunk 1 above)
Comment (guoguojin) [reply datanode/gateway/graphql/schema.graphql:478]: There appears to be a lot of these docstrings where the first word in the sentence is not capitalised. I'll check and make sure the others are updated too.

**Review by ze97286 — APPROVED** [review]

**Review by candida-d — APPROVED** [review]
